docs/walkthroughs/download-artifact-to-device.adoc

```diff
@@ -0,0 +1,59 @@
+[[mobile-client-builds]]
+= Mobile Client Builds
+
+== Open Mobile Client Builds
+To download artifact you need navigate to Overview -> Mobile Clients -> <your-mobile-client>
+
+image::images/mobile-client-overview.png[openshift-overview-tab]
+
+Then go to Builds tab to open mobile client builds
+
+image::images/mobile-client-builds-tab.png[openshift-mobile-client-builds-tab]
+
+== Start Mobile Client Build
+As you see that there are not builds currently available
```

Comment:
As you see that there are not builds currently available -> As you see there are no builds currently available

Reply:
do we need this "Start Mobile Client Build" section? We have whole document which is dedicated  in explaining on how to start a build